catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlResult.java

```diff
@@ -98,6 +100,24 @@
 
   private boolean isResourceLocal;
 
+  public CqlResult(CqlResult result, List<Map<String, String>> transformers) {
+    this.hasThumbnail = result.getHasThumbnail();
+    this.isResourceLocal = result.getIsResourceLocal();
+    this.distance = result.getDistance();
+    this.relevance = result.getRelevance();
+    this.metacard = result.getMetacard();
+    this.actions =
+        result
+            .getActions()
+            .stream()
+            .map(
+                cqlAction ->
```

Comment:
❓Should this parameter be named `action`? If this was a `CqlAction` then it would already have a display name, right?

Reply:
This is a very good point. At this point it is not a `CqlAction` yet